types.go

```diff
@@ -153,3 +157,14 @@ type Currency struct {
 	Precision int
 	Symbol    string
 }
+
+// RTAUStatus represents the Real Time Account Updater response from a processor, if applicable
+type RTAUStatus string
+
+const (
+	RTAUStatusUnknown      RTAUStatus = "Unknown"
+	RTAUStatusCardChanged  RTAUStatus = "CardChanged"
+	RTAUStatusCardExpired  RTAUStatus = "CardExpiryChanged"
+	RTAUStatusCloseAccount RTAUStatus = "CloseAccount"
+	RTAUExpiryTimeFormat = "1/2006"
```

Comment:
where do we use this?

Reply:
when we get a new expiry date, we need to parse it correctly and update our records; each processor treats expiry dates differently (i.e 1/2006 vs 1-2006 vs 01-06 etc) so creating an RTAU expiry standard that will be our internal representation of expiry updated through RTAU